third_party/move/move-vm/types/src/values/value_prop_tests.rs

```diff
@@ -12,13 +11,5 @@ proptest! {
         let blob = value.simple_serialize(&layout).expect("must serialize");
         let value_deserialized = Value::simple_deserialize(&blob, &layout).expect("must deserialize");
         assert!(value.equals(&value_deserialized).unwrap());
-
-        let move_value = value.as_move_value(&layout);
```

Comment:
Why are we removing this?

Reply:
As far as I understand, we have `Value` for runtime and `MoveValue` used as a core type (not sure what for but its more stable I was told before I think). Here the idea was to have an identical value in 2 representations and check it is the same, maybe I should have kept this :)

Reply:
Let me make `as_move_value` private and test only. The code there is a bit ugly, so many panics....